bci_tester/data.py

```diff
@@ -1076,7 +1143,7 @@ def create_BCI(
     custom_entry_point="/bin/bash",
 )
 
-VALKEY_CONTAINERS = [
+APP_VALKEY_CONTAINERS = [
```

Comment:
Is this renaming really necessary? It causes more diff changes, and no other variables seem to use this `APP_` prefix.

Reply:
Removed this name change